sdk/keyvault/azure-security-keyvault-administration/src/main/java/com/azure/security/keyvault/administration/implementation/models/Permission.java

```diff
@@ -0,0 +1,117 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.security.keyvault.administration.implementation.models;
+
+import com.azure.core.annotation.Fluent;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.List;
+
+/** The Permission model. */
+@Fluent
+public final class Permission {
```

Comment:
As mentioned above, classes in this folder were generated with AutoRest, so this name comes directly from the RBAC swagger. According to @jianghaolu, the use of custom directives such as [these](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/search/azure-search-documents/swagger/readme.md#set-odatametadata-accept-header-in-operations) is always discouraged, which is why I haven't added any. We can do it if absolutely required. Also, there is a new feature in the works that will allow us to modify the output Java code directly, but it might take a while to be released.

Reply:
After a brief chat with @srnagar, it seems that directives are ok but transformations are the issue. We can easily rename this.